lib/collection.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const BaseModel = require('./base');
+
+class CollectionModel extends BaseModel {
+    /**
+     * Construct a CollectionModel object
+     * @method constructor
+     * @param {Object} config
+     * @param {Object} config.datastore     Object that will perform operations on the datastore
+     * @param {Number} config.userId        The ID of the associated user
+     * @param {String} config.name          The collection name
+     * @param {String} config.description   The collection description
+     * @param {Array}  config.pipelineIds   The ids of the pipelines associated with this collection
```

Comment:
https://github.com/screwdriver-cd/data-schema/blob/master/models/collection.js
`description` and `pipelineIds` are optional for `create` based on your data-schema. But `pipelineIds` is required for `get`. So if user don't pass `pipelineIds` in when they create the collection, you should give it a default value as empty array either here or in the api. Otherwise there will be err when you try to get a collection without value for pipelineIds